### ISSUE TITLE
Prevents battleground scoreboard default keybind when awaiting resurrection

### ISSUE THREAD

**Issue by AlexFolland** [N0ich/dialogkey-DF#42]
When awaiting resurrection in a battleground, the default keybind for the battleground scoreboard of Shift+Space doesn't work, and DialogKey just spams "[DialogKey] This dialog cannot be clicked by DialogKey. Sorry!".

I was not trying to cancel the resurrection; just trying to view the battleground scoreboard.

Please make DialogKey not active at all and not overriding the Shift+Space keybind while the battleground resurrection dialog is visible.

**Comment by FuriousProgrammer** [comment]
This issue is actually already solved within the addon's options, although looking at it again right now it could definitely be organized better.

Under the Priority settings, either disable "Capture Broken Inputs" or enable "Ignore DialogKey with Modifiers".

Also, I'd highly recommend you change from using Spacebar to literally any other key! I personally use F, overlapping blizzard's interact action keybind.

**Comment by AlexFolland** [comment]
You use qwerty F to open the battleground scoreboard?  That's my right movement key, so the suggestion doesn't work for me.

Yes, "Capture Broken Inputs" is called `showErrorMessage` in the code, so I didn't realize it was connected.  I thought it was very weird that the option to show error messages was also changing the interaction logic, when it should only determine whether to show the error message.  My pull request solved that without me realizing that the intention for the option had already been changed retroactively without renaming the database variable.  Thanks for pointing out that the functionality already exists.

I think it would be good to make this particular interaction work by default, since it affects anyone using the default DialogKey and default battleground scoreboard keybinds, when it didn't for most of the previous years using DialogKey.  There could be many people who uninstalled DialogKey because they encountered this and aren't programmers like me who can fix it for themselves.

**Comment by FuriousProgrammer** [comment]
> You use qwerty F to open the battleground scoreboard? That's my right movement key, so the suggestion doesn't work for me.

Ah, no, I meant for the DialogKey keybind. I believe I still have Space+Shift as my default for the battleground scoreboard.

> My pull request solved that without me realizing that the intention for the option had already been changed retroactively without renaming the database variable.

Yeah, a handful of the options are like that. Originally the option /only/ controlled the error message itself, but I found that doing so without also consuming the input is bad UX. It's unfortunately not trivial to simply change the name of the option in the Ace3 database -- doing so just /creates a new option/ in the addon and ignores the old one, effectively resetting the setting to its default for everyone.

> I think it would be good to make this particular interaction work by default

Good defaults are hard! In this specific case the better option is to default-enable the "Ignore Modifier Keys" setting, I can't remember why we chose to have that off by default in the first place honestly.

That said, it's really not my problem if people find DialogKey doing something they don't like and don't think to check and explore the in-game options. As I said above, I admit that the settings could be laid out better (and the custom blacklist, while working in code, doesn't have an in-game configuration yet!)
